### PR TITLE
Add chat extension settings

### PR DESCRIPTION
**Pull request by Jalepeno112**
Allow support for [chat extensions](https://developers.facebook.com/docs/messenger-platform/design/guides/chat-extensions) which was revealed last week at F8.  This pull request adds 3 new controls into the UI to add and remove a chat extension for a Messenger app.

I have not yet been able to view my chat extension in Messenger, but doing requests on my app reveals that the configuration is being setup correctly.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -104,7 +104,10 @@ module.exports = {
     autoResponse: { type: 'string', required: false, default: 'Hello!' },     // deprecated
     autoResponseOption: { type: 'string', required: false, default: 'noResponse' },
     autoResponseText: { type: 'string', required: false, default: 'Hello, human!' },
-    autoResponsePostback: { type: 'string', required: false, default: 'YOUR_POSTBACK' }
+    autoResponsePostback: { type: 'string', required: false, default: 'YOUR_POSTBACK' },
+    chatExtensionHomeUrl: {type: "string", required: false, default: ''},
```
Comment (slvnperron) [inline src/index.js]: Could you please update the spacing here? We have yet to add a `.eslintrc` file 😓

Comment (Jalepeno112) [inline src/index.js]: Not a problem.  I'll fix that up and rebase

**Comment by Jalepeno112** [comment]
Hey @slvnperron, just wanted to check that you are seeing the updated pull request on your end.

The chat extension also started showing up in my Messenger app over the weekend, so I can finally verify that this implementation works!

**Comment by slvnperron** [comment]
Thanks, I indeed haven't seen the update! Let me know ow when this is working and I'll got it a go!

Btw it would be nice to be able to host extensions directly from Botpress (I.e have react views served from the bot directly). We're thinking about coding a small separate module for that.

**Comment by slvnperron** [comment]
Sorry for the typos, I should double read before posting. I meant: tell me when it's ready, I'll got it a go!

**Comment by Jalepeno112** [comment]
So the changes are definitely there.  There was a merge conflict after the last pull request I did, but I fixed that up too.  Let me know if you still aren't seeing it!

**Comment by Jalepeno112** [comment]
Awesome! Thanks!

**Comment by slvnperron** [comment]
@Jalepeno112 so sorry for the delay in merging those PRs. I didn't have notifications turned on for this repo :(

Great work on this by the way!!

**Comment by Jalepeno112** [comment]
No worries. Figured you were busy and this is isn't exactly a critical
feature. Thanks for the help!

**Pull request merged**